packages/components/src/box-control/utils.ts

```diff
@@ -250,6 +222,9 @@ export function applyValueToSides(
 	newValue?: string,
 	sides?: BoxControlProps[ 'sides' ]
 ): BoxControlValue {
+	deprecated( 'applyValueToSides', {
+		since: '6.8',
+	} );
```

Comment:
I'd personally be OK with removing this API, since it doesn't seem to be used at all, and it's been exported as experimental:

https://wpdirectory.net/search/01JEQVEXRM158GH2KVTS4XX9HJ

If we don't want to remove it immediately, perhaps we can at least specify a due version in which we can actually remove it.

I'm happy to move this to a separate PR, so we can discuss it and it gets proper attention.

Reply:
Agreed, it's prefixed so it's one of these "experimental but stable APIs" which means we should consider its usage to make the right call and give that it's not really used, it should be fine to remove. I'm keeping it around for a couple releases just to be on the safe side.